geolib/models/dsettlement/templates/input.sli.j2

```diff
@@ -276,22 +276,22 @@ D-Settlement={{ version.d__settlement }}
 
 [CALCULATION OPTIONS]
 {% if calculation_options is mapping -%}
-{{ calculation_options.precon_pressure_within_layer.value }} : Precon. pressure within a layer = {{ calculation_options.precon_pressure_within_layer.name }}
+{{ calculation_options.precon_pressure_within_layer.value }} : Precon. pressure within a layer = {{ {0: 'Constant (constant in the layers)', 1: 'Constant, correction at t=0 [days]', 2: 'Constant, correction at every step', 3: 'Variable (parallel to eff. stress)', 4: 'Variable, correction at t=0 [days]',  5: 'Variable, correction at every step'}[calculation_options.precon_pressure_within_layer.value] }}
 {{ calculation_options.is_imaginary_surface.value }} : Imaginary surface = {{ calculation_options.is_imaginary_surface.name }}
 {% if calculation_options.imaginary_surface_layer is not none %}
```

Comment:
I still think the if statement should be removed, because the next line must always be written. You say it will never happen, but that it is none, but then the if statement is useless anyway.

Reply:
No, the line giving the layer index of the imaginary surface is written only if is_imaginary_surface is true. See in bm1-1.sli.

Reply:
oh